.github/workflows/cicd-1-pull-request.yaml

```diff
@@ -94,9 +94,9 @@ jobs:
     secrets: inherit
   build-image-stage: # Recommended maximum execution time is 3 minutes
     name: Image build stage
-    needs: [metadata, commit-stage, test-stage]
-    uses: NHSDigital/dtos-devops-templates/.github/workflows/stage-3-build-images.yaml@main
-    if: needs.metadata.outputs.does_pull_request_exist == 'true' || github.ref == 'refs/heads/main' || (github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'reopened'))
+    # needs: [metadata, commit-stage, test-stage]
+    uses: NHSDigital/dtos-devops-templates/.github/workflows/stage-3-build-images.yaml@feat/DTOSS-5834-PostgreSQL-module-for-nft-and-int
```

Comment:
Is this an intentional commit? I don't think we would want this in main branch would we?